build.rs

```diff
@@ -1,3 +1,4 @@
+#[cfg(not(docsrs))]
 use auto_generate_cdp::init;
 
 fn main() {
```

Comment:
The whole crate depends on `protocol.rs` in order to compile if we do this the `protocol.rs` file wont be generated

Reply:
Damn - afaik the only other option is to download a local version of the file and use that - I presume that will require repo changes in `auto_generate_cdp`

Reply:
> Damn - afaik the only other option is to download a local version of the file and use that - I presume that will require repo changes in `auto_generate_cdp`

Let me see if i can include the file based on if its building inside docs.rs